backend/Application/Api/GraphQL/Import/AccountChangeCalculator.cs

```diff
@@ -75,7 +75,7 @@ public AccountTransactionRelation[] GetAccountTransactionRelations(TransactionPa
 
         return result;
     }
-
+    
```

Comment:
can you please try to avoid having these kinds of whitespace changes. They just clutter the PR and make it more difficult to review.

Reply:
Sure